Common/Util/CurrencyPairUtil.cs

```diff
@@ -192,12 +192,37 @@ public enum Match
         /// <returns>The <see cref="Match"/> member that represents the relation between the two pairs</returns>
         public static Match ComparePair(this Symbol pairA, string baseCurrencyB, string quoteCurrencyB)
         {
-            if (pairA.Value == baseCurrencyB + quoteCurrencyB)
+            bool isThereAnyStableCoin = false;
+            // Check for a stablecoin between quote currencies
+            // First, make sure the pair A contains B base currency
+            if (pairA.Value.Contains(baseCurrencyB))
+            {
+                var pairAQuoteCurrency = pairA.Value.Substring(pairA.Value.IndexOf(baseCurrencyB) + baseCurrencyB.Length);
```

Comment:
I think would be better to use `DecomposeCurrencyPair` to get the quote currency of A, same bellow for the base currency, that method already knows how to do it correctly.